src/ensembl/src/content/app/entity-viewer/state/gene-view/transcripts/geneViewTranscriptsSlice.ts

```diff
@@ -231,6 +269,12 @@ type UpdateSortingRulePayload = {
   sortingRule: SortingRule;
 };
 
+type UpdateFilterPanelOpenPayload = {
```

Comment:
`UpdateFilterPanelOpenPayload` -> `UpdateFilterOpenPayload`?

Reply:
I am still unsure how we refer to that box that contains filter and sorting options. Looks like it's called "filter panel" here. Is there any reason to change this?

Reply:
I just thought that the word `panel` here is simply not required?
The action creator is called `updateFilterPanel` but all we do there is toggle the open state.

Reply:
> The action creator is called updateFilterPanel but all we do there is toggle the open state.

Yes, but we are toggling the open state of the panel 😃 🤷‍♂️

Reply:
Comparing `setFilterPanel`, `resetFilterPanel` & `UpdateFilterPanelOpenPayload`, I wonder why the type definition  
alone talks about the `Open` state when the other two doesn't?

Reply:
I agree that `UpdateFilterPanelOpenPayload` type can be renamed to `UpdateFilterPanelPayload`.